charts/pega/README.md

```diff
@@ -480,6 +480,17 @@ Parameter           | Description    | Default value
 `hpa.enableCpuTarget` | Set to true if you want to enable scaling based on CPU utilization or false if you want to disable it | true
 `hpa.enableMemoryTarget` | Set to true if you want to enable scaling based on memory utilization or false if you want to disable it (Pega recommends leaving this disabled) | false
 
+### Setting a Pod Disruption Budget (PDB)
+You can configure a Kubernetes `PodDisruptionBudget` on your tier by setting `pdb.enabled` to `true`.  For more details, see the Kubernetes 
```

Comment:
for line 483, can we add a context and not rely on the user to put together performance and a "disruption budget" as an equivalence?
To safeguard your deployment by limiting the number of Pods running your Pega Platform application that can go down for planned "disruptions", Pega allows you to enable a Kubernetes `PodDisruptionBudget`. For more details, see the Kubernetes...